src/main/java/core/selenium/utils/ConfigFileReader.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * ConfigFileReader class
+ *
+ * v1.0
+ *
+ * 09-17-2019
+ *
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package core.selenium.utils;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * ConfigFileReader class.
+ *
+ * @author John Salazar Pinto
+ */
+public class ConfigFileReader {
+    private Properties properties;
+    private final String propertyFilePath = "PivotalTracker.properties";
+
+    /**
+     * Creates a new property that read the datas from an external file.
+     */
+    public ConfigFileReader() {
+        BufferedReader reader;
+        try {
+            reader = new BufferedReader(new FileReader(propertyFilePath));
+            properties = new Properties();
+            try {
+                properties.load(reader);
+                reader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
```

Comment:
maybe you put a Throw exception or this catch

Reply:
I change the exception